crate/pages/events/docker-meetup-berlin-april-2015.html

```diff
@@ -0,0 +1,11 @@
+title: Docker Meetup Berlin
```

Comment:
maybe add the month to the title for better search results?
e.g. Docker Meetup Berlin (April 2015)